src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.ui.Span;
+import duke.ui.UI;
+
+import duke.storage.FileManagement;
+
+import duke.task.TaskList;
+
+import duke.exceptions.DukeException;
+
+/**
+ * Duke is a personal assistant bot that helps user to track tasks.
+ *
+ * @author Andre Lin HuiKai
+ * @version CS2103T AY22/23 Semester 2
+ */
+public class Duke {
+    private FileManagement fileManager;
```

Comment:
Great job in following the coding standard for your duke class ! 👍

Reply:
Thanks!